src/ui/public/filters/_prop_filter.js

```diff
@@ -37,15 +37,22 @@ export function propFilter(prop) {
    *   - Can be also an array, a single value as a string, or a comma-separated list of items
    * @return {array} - the filtered list
    */
-  return function (list, filters) {
-    if (!filters) return filters;
+  return function (list, filters = []) {
+    if (filters.length === 0) {
```

Comment:
documentation for this says `filters` can be a single function. `.length` of a function returns number of arguments the function expects, so could be `0`

Reply:
Good catch!